.github/workflows/editorconfig.yml

```diff
@@ -13,12 +13,17 @@ jobs:
       with:
         fetch-depth: 0
     - uses: technote-space/get-diff-action@v2.0.3
-    - name: Checking EditorConfig
+    - name: fetch editorconfig-checker
       env:
         VERSION: "2.1.0"
         OS: "linux"
         ARCH: "amd64"
+        ECC_URL: "https://github.com/editorconfig-checker/editorconfig-checker/releases/download"
       run: |
-        curl -sSf -O -L -C - https://github.com/editorconfig-checker/editorconfig-checker/releases/download/$VERSION/ec-$OS-$ARCH.tar.gz && \
+        curl -sSf -O -L -C - $ECC_URL/$VERSION/ec-$OS-$ARCH.tar.gz && \
         tar xzf ec-$OS-$ARCH.tar.gz && \
-        ./bin/ec-$OS-$ARCH -disable-indentation ${{ env.GIT_DIFF }}
+        mv ./bin/ec-$OS-$ARCH ./bin/editorconfig-checker
+    - name: Checking EditorConfig
+      run: |
+        ./bin/editorconfig-checker -disable-indentation \
```

Comment:
That line break does not feel necessary, even with the line below your below 80 characters.

Reply:
It's for when multiple files are edited so that they are all after the break rather than being cut off.

Reply:
Okay, works for me.